src/main/scala/is/hail/methods/SampleQC.scala

```diff
@@ -156,19 +156,20 @@ class SampleQCCombiner extends Serializable {
 }
 
 object SampleQC {
-  def results(vds: GenericDataset): Array[SampleQCCombiner] = {
+  def results[RPK, RK, T >: Null](vds: VariantSampleMatrix[RPK, RK, T]): Array[SampleQCCombiner] = {
```

Comment:
rename `vds` to `vsm` (or `vkds` if you want to consistently use that abbreviation for variant-keyed `vsm`.)